vasyl_varyshchuk/Task_2/retry_decorator.py

```diff
@@ -0,0 +1,31 @@
+"""
+This module implements a retry decorator with ability to pass a parameter into decorator
+"""
+
+
+def decorator(retries=None):
+
+    def my_decorator(func):
+
+        def func_wrapper():
+
+            i = 0
+            while i < retries:
```

Comment:
The same situation as in my previous comment.

Reply:
As was mentioned above, this mistake was fixed.

Reply:
I can't see any changes in this block of code.